scripts/save_batches.py

```diff
@@ -20,21 +20,23 @@
 import shutil
 import sys
 
+import dask
+
+dask.config.set(scheduler="single-threaded")
```

Comment:
Is this needed? Could it instead go in the relevant part of the datapipe where the values are computed.  Like using the `.compute(scheduler="single-threaded")`